composer.lock

```diff
@@ -1789,16 +1789,16 @@
         },
         {
             "name": "silex/silex",
-            "version": "v2.2.2",
+            "version": "v2.2.4",
```

Comment:
I don't think that such updates should happen as a by-product of a, AFAICT, completely unreleated front-end story. Accidentally committed?